src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -186,3 +186,79 @@ Potrebbero esservi delle issue aperte all'interno dell'_ITS_ che non hanno un co
+
+I ruoli assunti sono i seguenti:
+
+==== Responsabile  
+
+        Figura chiave nell'ambito della gestione di progetti. Ha la responsabilità sul controllo, coordinamento e relazioni esterne.
+        I suoi compiti sono:
+
+         - Gestione della comunicazione con la Proponente (si fa uso della piattaforma Element);
+         - Preparazione dell'ordine del giorno per la successiva riunione, anche sulla base
+           dei punti individuati dagli altri componenti;
+         - Redazione dei verbali interni ed esterni;
+         - Stesura e e avanzamento del documento “Piano di progetto”;
+         - Creazione dei diari di bordo;
+         - Scaricamento dei verbali esterni convalidati e immagazzinamento dentro la cartella IncontriAziende/VerbaliEsterni_Convalidati su Google Drive.
+
+==== Amministratore 
+        
+        Figura con l'incarico di controllo e amministrazione dell'ambiente di lavoro.
+
+        I suoi compiti sono:
+         - Creazione delle issue e loro assegnazione ai membri;
+         - Stesura e avanzamento del documento "Norme di Progetto" (way of working);
+         - Automatizzazione dell'ambiente di lavoro con script specifici;
+         - Gestione del versionamento dei documenti.
+
+==== Analista
+     
+     Figura professionale che si occupa di analizzare, comprendere e definire i requisiti e le specifiche di un sistema software prima che venga sviluppato. Questa figura svolge un ruolo fondamentale nel processo di sviluppo del software, contribuendo a garantire che il prodotto finale soddisfi le esigenze e le aspettative degli utenti e della Proponente.
+
+     I suoi compiti:
+        - Studio del contesto applicativo e relativa complessità;
+        - Raccolta dei Requisiti;
+        - Stesura del documento Analisi dei Requisiti;
+        - Creazione diagrammi UML;
+
+==== Progettista
+
+        Figura professionale che si occupa della progettazione architetturale e struttura di un sistema. Si concentra sulla definizione di come il sistema dovrebbe essere organizzato.
+
+        I suoi compiti sono:
+
+        - Scelta degli aspetti tecnici e tecnologici;
+        - Progettazione architetturale che miri all'economicità e alla manutenibilità del sistema;
+        - Ottimizzazione delle prestazioni usando algoritmi efficienti e gestione memoria;
+        - Gestione dei rischi: cerca di mitigare problemi che possono sorgere durante lo sviluppo;
+        - Redazione della Specifica tecnica e di una parte del documento Piano di Qualifica.
+
+==== Programmatore
+
+        Figura professionale che si occupa della codifica del sistema. 
+
+        I suoi compiti:
+
+        - Ampliamento delle Specifiche Tecniche;
+        - Scrittura codice pulito;
+        - Redazione di test per validare e verificare il codice;
+        - Redazione del Manuale Utente.
```

Comment:
Il Programmatore è il professionista incaricato di trasformare le specifiche tecniche in codice eseguibile, garantendo un'implementazione efficiente e accurata delle funzionalità richieste dal progetto.
I suoi compiti:

- Traduzione delle specifiche tecniche in codice funzionante;
- Scrittura di codice chiaro, leggibile e mantenibile;
- Creazione di test per la verifica del software;
- Ampliamento delle Specifiche Tecniche conforme alle esigenze del progetto.
- Risoluzione di bug e problemi di performance;
- Realizzazione del Manuale Utente;
- Collaborazione con il team per l'integrazione del codice e il mantenimento della coerenza del progetto.